### DISCUSSION TITLE
Follow CA1825 Use Array<T>.Empty in all instances

### DISCUSSION THREAD

**Discussion by Mrnikbobjeff** [dotnet/maui#22613] [Ideas]
### Description

Enabling the analyzer would allow the automatic detection of code such as [this](https://github.com/search?q=repo%3Adotnet%2Fmaui%20new%20int%5B0&type=code). It is a automatable fix with only positive impact.
See this [docs link](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1825).

### Public API Changes

None

### Intended Use-Case

Allocation behaviour improvement / Code quality

**Comment by MartyIX** [comment]
It's enabled here as far as I can tell: https://github.com/dotnet/maui/blob/1e74e8f6d3a1b0d849ea94d10fe42db3d3156a7a/.editorconfig#L32
